pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -33,7 +33,6 @@ type ofFlowAction struct {
 // Drop is an action to drop packets.
 func (a *ofFlowAction) Drop() FlowBuilder {
 	a.builder.Drop()
-	a.builder.isDropFlow = true
```

Comment:
Why is this deleted?

Reply:
We don't consume `isDropFlow` after this patch, and I also removed the field.